<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgVoiceChange</class>
 <widget class="QDialog" name="DlgVoiceChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>变声</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgVoiceChange {
	background: rgb(255,255,255);
}
QListWidget {
	border: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="userListWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
